specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/AppServiceEnvironments.json

```diff
@@ -1122,6 +1122,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/hostingEnvironments/{name}/upgrade": {
+      "post": {
+        "tags": [
+          "AppServiceEnvironments"
+        ],
+        "summary": "Initiate an upgrade of an App Service Environment if one is available.",
+        "description": "Description for Initiate an upgrade of an App Service Environment if one is available.",
+        "operationId": "AppServiceEnvironments_Upgrade",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the App Service Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "testNotification",
+            "in": "query",
+            "description": "Whether the request is to send an upgrade test notification instead of starting an upgrade.",
+            "type": "boolean"
+          },
```

Comment:
The parametres need to be in the body , not the query as explained here : 

[https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/proxy-api-reference.md#resource-action-requests](https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/proxy-api-reference.md#resource-action-requests)